frontend/src/components/Issue/TaskCheckRunPanel.vue

```diff
@@ -149,6 +149,7 @@ export default defineComponent({
         case SQLReviewPolicyErrorCode.INDEX_NAMING_DISMATCH:
         case SQLReviewPolicyErrorCode.UK_NAMING_DISMATCH:
         case SQLReviewPolicyErrorCode.FK_NAMING_DISMATCH:
+        case SQLReviewPolicyErrorCode.PK_NAMING_DISMATCH:
```

Comment:
After looking up the dictionary, I think the word is "mismatch" not "dismatch". You can fix these words in a follow-up PR.

Reply:
Just fix this in this PR